modules/Vtiger/helpers/SocialMedia.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * SocialMedia Helper.
+ *
+ * @copyright YetiForce Sp. z o.o
+ * @license   YetiForce Public License 3.0 (licenses/LicenseEN.txt or yetiforce.com)
+ * @author    Arkadiusz Adach <a.adach@yetiforce.com>
+ */
+class Vtiger_SocialMedia_Helper extends \App\Base
+{
+	/**
+	 * @var \Abraham\TwitterOAuth\TwitterOAuth
+	 */
+	private $twitterConnection;
+
+	/**
+	 * Get all social media accounts.
+	 *
+	 * @return \Generator
+	 */
+	public static function getSocialMediaAccount()
+	{
+		$dataReader = (new \App\Db\Query())
+			->select(['columnname', 'tablename'])
+			->from('vtiger_field')
+			->where(['uitype' => 313])
+			->andWhere(['presence' => [0, 2]])
+			->createCommand()
+			->query();
+		while (($row = $dataReader->read())) {
+			$dataReaderAccounts = (new \App\Db\Query())
+				->select([$row['columnname']])
+				->from($row['tablename'])
+				->andWhere(['not', [$row['columnname'] => null]])
+				->andWhere(['not', [$row['columnname'] => '']])
+				->createCommand()
+				->query();
+			while (($rowTwitter = $dataReaderAccounts->read())) {
```

Comment:
$dataReaderAccounts = (new \App\Db\Query())
				->select([$row['columnname']])
				->from($row['tablename'])
				->andWhere(['not', [$row['columnname'] => null]])
				->andWhere(['not', [$row['columnname'] => '']])
				->column();